src/libraries/System.Reflection/tests/ParameterInfoTests.cs

```diff
@@ -83,7 +83,16 @@ public void HasDefaultValue_broken_on_NETFX(Type type, string name, int index, b
         public void HasDefaultValue_ReturnParam()
         {
             ParameterInfo parameterInfo = GetMethod(typeof(ParameterInfoMetadata), "Method1").ReturnParameter;
-            Assert.True(parameterInfo.HasDefaultValue);
+
+            if (PlatformDetection.IsNativeAot)
+            {
+                // it's hard coded as true
+                Assert.True(parameterInfo.HasDefaultValue);
```

Comment:
Why did not change the hard coded value? I think this should behave the same for native AOT cc @jkotas